herbert_core/classes/instrument_classes/@Goniometer/private/check_and_set_uv_.m

```diff
@@ -5,7 +5,7 @@
 if isnumeric(val)
     if numel(val)==3
         cand = val(:)'; % make it row vector
-    elseif numel(val) == 1
+    elseif isscalar(val)
```

Comment:
I am presuming that this is the actual fix but I have no idea why it works. To ensure that future development in this area does not accidentally revert this change, please add in the comments a full explanation of what it does. Thanks

Reply:
`isscalar` is the new Matlab recommended way of dealing with expressions like `row 8`. I am changing code to Matlab recommended as I go through code, but the actual solution is in other file.